projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -260,6 +261,15 @@ public static function register_rest_endpoints() {
 				'callback' => __CLASS__ . '::api_clear_scan_cache',
 			)
 		);
+
+		register_rest_route(
+			'jetpack-protect/v1',
+			'ignore-threat',
+			array(
+				'methods'  => \WP_REST_SERVER::EDITABLE,
```

Comment:
We should probably add a capabilities check here to make sure the user is authenticated.